scripts/mirror.test.js

```diff
@@ -0,0 +1,141 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+'use strict';
+
+const assert = require('assert').strict;
+
+const { browsers } = require('..');
+const { getMatchingBrowserVersion } = require('./mirror');
+
+describe('mirror', function () {
+  const match = (source_browser, source_version, dest_browser) => {
+    const source_release = browsers[source_browser].releases[source_version];
+    return getMatchingBrowserVersion(dest_browser, source_release);
+  };
+
+  it('chrome_android', function () {
+    assert.equal(match('chrome', '1', 'chrome_android'), '18');
+    assert.equal(match('chrome', '2', 'chrome_android'), '18');
+    assert.equal(match('chrome', '17', 'chrome_android'), '18');
+    assert.equal(match('chrome', '18', 'chrome_android'), '18');
+    assert.equal(match('chrome', '19', 'chrome_android'), '25');
+    assert.equal(match('chrome', '24', 'chrome_android'), '25');
+    assert.equal(match('chrome', '25', 'chrome_android'), '25');
+    assert.equal(match('chrome', '26', 'chrome_android'), '26');
+  });
+
+  it('edge', function () {
+    assert.equal(match('chrome', '1', 'edge'), false);
+    assert.equal(match('chrome', '27', 'edge'), false);
+    assert.equal(match('chrome', '28', 'edge'), '79');
```

Comment:
This doesn't make sense and is because early versions of Chrome had WebKit as the engine, not Blink. It isn't sensible to infer Edge 79 from Edge 28, but not from Edge 27. We should treat all Chrome versions <79 the same in terms of what we infer for Edge.

Reply:
Edge mirroring doesn't use `getMatchingBrowserVersion()` and instead uses custom logic that infers ranges based on the original Edge data.  I imagine we'll simplify the code for build-time mirroring, but we should probably test `bumpVersion()` instead of `getMatchingBrowserVersion()`.

Reply:
I'll remove this part of the test then.